### ISSUE TITLE
Rank should not be required for Prospect on New Person

### ISSUE THREAD

**Issue by VanAuken** [ReadyResponder/ReadyResponder#712]
When making a New Person, people are often entered as Prospect or Applicant.  Rank is required in order to save, and currently we are using 'unknown' to meet requirements.

Accurate logic would not require a rank for 'prospect' or 'applicant'.  A rank is only needed for people who have joined.  This would also require a blank or null choice for rank.

**Comment by yasirazgar** [comment]
I can look into this.

**Comment by houhoulis** [comment]
@VanAuken ,

Considering a person might have a Status such as "Applicant", I approve of removing the requirement that a "Rank" (title) be entered for them.  However, I'm not certain, but this issue might be requesting to require a "Rank" for certain statuses but not others.  I'm not in favor of hard-coding that level of logic into the app; could we just make "Rank" optional for all people?

I'm not in favor of hard-coding 'Rank required for Statuses X, Y, and Z' logic because I'm not sure that all organizations will have the same idea about which Statuses should require a Rank.  Until we have more configurability built into the app, I'd rather avoid such detailed restrictions.

Can we make "Rank" optional for all people?

**Comment by kgf** [comment]
I agree with Chris, both about not trying to hard code a matrix, and not requiring title/rank for anyone at this time.

**Comment by karmenlu** [comment]
I would like to work on/claim this issue. @kgf

**Comment by kgf** [comment]
Thanks, @klu17
